MaxText/pyconfig.py

```diff
@@ -134,7 +136,16 @@ def get_individual_scales(scale):
   layer_scale = base_scale
   return emb_scale, num_head_scale, mlp_dim_scale, layer_scale
 
+def get_batch_sizes(per_device_batch_size):
```

Comment:
`calculate_global_batch_sizes` since you're deriving this one. But totally a nit, even nittier than my other nits and I wouldn't make you redo just for that.